pom.xml

```diff
@@ -131,13 +145,60 @@
                 <artifactId>jquery</artifactId>
                 <version>2.2.4</version>
             </dependency>
+            <!-- Service Locator Provider -->
+            <dependency>
+                <groupId>${serviceLocator.provider.groupName}</groupId>
+                <artifactId>${serviceLocator.provider.artifactName}</artifactId>
+                <version>${serviceLocator.provider.artifactVersion}</version>
+            </dependency>
         </dependencies>
     </dependencyManagement>
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-
         <lagom.version>1.3.4</lagom.version>
+        <!--
         <conductr.lib.name>lagom1-java-conductr-bundle-lib_2.11</conductr.lib.name>
+        -->
     </properties>
+
+
+    <repositories>
+      <repository>
+        <id>akka-dns</id>
+        <name>akka-dns</name>
+        <url>https://dl.bintray.com/hajile/maven</url>
+        <snapshots>
+          <enabled>false</enabled>
+        </snapshots>
+      </repository>
+    </repositories>
+
+    <profiles>
+      <profile>
+        <id>conductr</id>
+        <properties>
+          <buildTarget>conductr</buildTarget>
+          <serviceLocator.provider.groupName>com.typesafe.conductr</serviceLocator.provider.groupName>
+          <serviceLocator.provider.artifactName>lagom1-java-conductr-bundle-lib_2.11</serviceLocator.provider.artifactName>
+          <serviceLocator.provider.artifactVersion>1.9.1</serviceLocator.provider.artifactVersion>
+        </properties>
+        <activation>
+          <activeByDefault>false</activeByDefault>
+        </activation>
+      </profile>
+
+      <profile>
+        <id>kubernetes</id>
+        <properties>
+          <buildTarget>kubernetes</buildTarget>
+          <serviceLocator.provider.groupName>com.lightbend</serviceLocator.provider.groupName>
+          <serviceLocator.provider.artifactName>lagom-service-locator-dns_2.11</serviceLocator.provider.artifactName>
+          <serviceLocator.provider.artifactVersion>1.0.2</serviceLocator.provider.artifactVersion>
+        </properties>
+        <activation>
+          <activeByDefault>true</activeByDefault>
```

Comment:
I was just wondering why Kubernetes is the default profile for maven and not for SBT. Wouldn't it be better to align?